br/pkg/lightning/backend/local/local.go

```diff
@@ -549,7 +549,7 @@ func NewBackend(
 		"lightning",
 		pdCli.GetServiceDiscovery(),
 		pdhttp.WithTLSConfig(tls.TLSConfig()),
-	).WithBackoffer(retry.InitialBackoffer(time.Second, time.Second, pdutil.PDRequestRetryTime*time.Second))
+	).WithBackoffer(retry.InitialBackoffer(time.Second, time.Second, pdutil.PDRequestRetryTime*time.Second)).WithTimeout(dialTimeout)
```

Comment:
I think the function name and comments should be clear about two points:

1. why we need to have another timeout since we already use context?
2. what's the behaviour of the context and this timeout is exceeded?